test/kubemark/common.sh

```diff
@@ -23,3 +24,27 @@ export PROJECT
 MASTER_NAME="${INSTANCE_PREFIX}-kubemark-master"
 MASTER_TAG="kubemark-master"
 EVENT_STORE_NAME="${INSTANCE_PREFIX}-event-store"
+
+RETRIES=3
+
+# arguments:
+# $1: name of operation to embed in an error message
```

Comment:
This seems unnecessary; why not just use `$2`?  It's extra flags in error messages, but those are often important to know about.